.github/workflows/nodejs.yml

```diff
@@ -55,3 +55,30 @@ jobs:
           install-command: npm ci --foreground-scripts
       - name: Lint
         run: npm run lint
+
+  validate-lockfile:
+    runs-on: ubuntu-latest
+    env:
+      FORCE_COLOR: 1
+    strategy:
+      matrix:
+        node-version: [18.x]
```

Comment:
this may benefit from running against multiple versions of `npm`, but it does not benefit from running against multiple versions of Node.js.

Reply:
then how about we remove the matrix setup? We're installing npm latest anyway